modelforge/curation/qm9_curation.py

```diff
@@ -373,16 +400,25 @@ def process(
             output_path=self.local_cache_dir,
             force_download=force_download,
         )
```

Comment:
is this function imported?

Reply:
Good catch.  it was not (and this part of the code was not being checked by CI because it takes too long).  All the more reason for the more thorough offline suite checking.